tests/foreman/ui/test_organization.py

```diff
@@ -289,3 +322,83 @@ def test_positive_download_debug_cert_after_refresh(session):
                 session.subscription.refresh_manifest()
     finally:
         entities.Subscription(organization=org).delete_manifest(data={'organization_id': org.id})
+
+
+@pytest.mark.tier2
+def test_negative_errata_view_organization_switch(
+    session, module_org, module_lce, module_repos_col, request, default_sat
+):
+    """Verify no errata list visible on Organization switch
+
+    :id: a5317b2d-2593-4bee-9b20-78066cf2964c
+
+    :Steps: Create an Organization having a product synced which contains errata.
+
+    :expectedresults: Verify that the errata belonging to one Organization is not
+                      showing in the Default organization.
+
+    :CaseImportance: Critical
+
+    :CaseLevel: Integration
+    """
+    rc = RepositoryCollection(repositories=[YumRepository(settings.repos.yum_3.url)])
+    rc.setup_content(module_org.id, module_lce.id)
+    with session:
+        assert (
+            session.errata.search(CUSTOM_REPO_ERRATA_ID, applicable=False)[0]['Errata ID']
+            == CUSTOM_REPO_ERRATA_ID
+        )
+        session.organization.select(org_name="Default Organization")
+        assert not session.errata.search(CUSTOM_REPO_ERRATA_ID, applicable=False)
+
+        @request.addfinalizer
+        def _cleanup():
```

Comment:
@shweta83 I think we should move this cleanup part in `module_repos_col` fixture itself.

Reply:
Agree. Moved this to fixture.